src/allotropy/parsers/lines_reader.py

```diff
@@ -8,6 +8,8 @@
 
 from allotropy.allotrope.allotrope import AllotropyError
 
+EMPTY_STR = r"^\s*$"
```

Comment:
`EMPTY_STR_PATTERN`? I'd expect an `EMPTY_STRING` constant to be equal to `""`.

Reply:
thanks - fixed in the latest push